packages/client-core/src/common/components/WarningMessage/WarningMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import InfoIcon from '@mui/icons-material/Info'
+import Button from '@mui/material/Button'
+import Dialog from '@mui/material/Dialog'
+import MuiDialogActions from '@mui/material/DialogActions'
+import MuiDialogContent from '@mui/material/DialogContent'
+import MuiDialogTitle from '@mui/material/DialogTitle'
+
+import styles from '../index.module.scss'
+import { BrowserMessages } from './BrowserMessages'
+
+const WarningMessage = ({ setIsHardwareAccelerationEnabled }) => {
+  const browser = {
+    isFirefox: navigator.userAgent.indexOf('Firefox') != -1,
```

Comment:
this probably is not good enough, we need to have a more robust way of detecting which browser is being used